frontend/src/app/_components/BugReportModal/BugReportModal.tsx

```
@@ -0,0 +1,371 @@
+    useShallow((state) => ({
+      activeModal: state.activeModal,
+      closeModal: state.closeModal,
+    })),
+  );
+
+  const isOpen = activeModal === MODAL_TYPES.BUG_REPORT;
+
+  const { contentRef, handleClose, handleBackdropClick } = useModalClose({
+    isOpen,
+    onClose: () => {
+      attachments.forEach((a) => URL.revokeObjectURL(a.preview));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)